crates/flox/src/commands/mod.rs

```diff
@@ -218,20 +218,21 @@ enum Commands {
     Additional(#[bpaf(external(additional_commands))] AdditionalCommands),
     Internal(#[bpaf(external(internal_commands))] InternalCommands),
 }
+///flox is a virtual environment and package manager all in one. With flox you create development environments that layer and replace dependencies just where it matters, making them portable across the full software lifecycle.
```

Comment:
```suggestion
/// flox is a virtual environment and package manager all in one. With flox you create development environments that layer and replace dependencies just where it matters, making them portable across the full software lifecycle.
```

Maybe I've read this too many times, but now I can't tell which of these two this is saying:
- environments layer, environments replace dependencies
- environments layer and replace dependencies (i.e. "layer and replace" are both referring to dependencies)

"replace dependencies" doesn't sit right with me for some reason. Maybe it sounds like we're doing something with existing libraries? If I'm a user and that's how I interpret it I would feel weird about a tool touching my existing libraries and stuff. Would something like "making the right dependencies available" be a reasonable replacement?

Reply:
CC @jennymahmoudi on the text. at least when spoken so far it's gone over well. we're saying replace because it will overlay any existing installs (i.e. your home brew installed python)